server/src/internalClusterTest/java/org/elasticsearch/search/aggregations/bucket/AdjacencyMatrixIT.java

```diff
@@ -280,7 +279,7 @@ public void testTooLargeMatrix() throws Exception {
 
         // Create more filters than is permitted by Lucene Bool clause settings.
         MapBuilder filtersMap = new MapBuilder();
-        int maxFilters = SearchModule.INDICES_MAX_CLAUSE_COUNT_SETTING.get(Settings.EMPTY);
+        int maxFilters = IndexSearcher.getMaxClauseCount();
```

Comment:
This number might be super high now, how slow is the test? Maybe we need to set an artificially low limit for this test to keep it reasonable?